understanding/20/name-role-value.html

```diff
@@ -50,8 +50,13 @@ <h2>Intent of Name, Role, Value</h2>
          focus. The focus state of a control can be programmatically determined, and notifications
          about change of focus are sent to user agents and assistive technology. Other examples
          of user interface control state are whether or not a checkbox or radio button has
```

Comment:
I think states should be plural here?
```suggestion
         of user interface control states are whether or not a checkbox or radio button has
```